### ISSUE TITLE
too much time spent in timers

### ISSUE THREAD

**Issue by poandrei** [intuitivelabs/calltr#9]
From intuitive-issues created by [poandrei](https://github.com/poandrei): intuitivelabs/intuitive-issues#261  

Timers should be high on the list of improvements.

profile on high traffic, sipcm only (without the beat sending events part):
```
Showing top 10 nodes out of 117
      flat  flat%   sum%        cum   cum%
    2260ms 17.68% 17.68%     2320ms 18.15%  runtime.adjusttimers
    2150ms 16.82% 34.51%     2370ms 18.54%  runtime.timeSleepUntil
    1900ms 14.87% 49.37%     1900ms 14.87%  runtime.futex
     490ms  3.83% 53.21%      490ms  3.83%  runtime.usleep
     430ms  3.36% 56.57%      480ms  3.76%  runtime.cgocall
     290ms  2.27% 58.84%      330ms  2.58%  github.com/intuitivelabs/sipsp.ParseNameAddrPVal
     250ms  1.96% 60.80%      250ms  1.96%  runtime.epollwait
     220ms  1.72% 62.52%     4040ms 31.61%  runtime.findrunnable
     180ms  1.41% 63.93%      180ms  1.41%  github.com/intuitivelabs/sipsp.skipToken
     170ms  1.33% 65.26%      240ms  1.88%  runtime.nanotime (inline)
```

![profile001](https://user-images.githubusercontent.com/44233689/110917982-d9b77780-831a-11eb-97f8-4419aa583650.png)

**Comment by poandrei** [comment]
results with scale 0.1 (10x speed-up), latest branch:
```
57664 pkts/s, sip udp ok: 44613/s, tcp ok: 936/s
         16535 REG/s, 1233 INV/s
         5806 events/s, rate blst 552/s
         120 call-start/s 754 reg-new/s  2224 reg-del/s !
          max active call-state: 66261 (4379 inv established, 54156 non inv, ...)
          max active reg-bindings: 23284
```
Profile:
```
Showing top 10 nodes out of 109
      flat  flat%   sum%        cum   cum%
    14.02s 35.83% 35.83%     14.23s 36.37%  runtime.adjusttimers
     5.23s 13.37% 49.19%      5.38s 13.75%  runtime.timeSleepUntil
     1.97s  5.03% 54.23%      2.07s  5.29%  runtime.cgocall
     1.24s  3.17% 57.40%      2.55s  6.52%  runtime.scanobject
     1.16s  2.96% 60.36%      1.32s  3.37%  github.com/intuitivelabs/sipsp.ParseNameAddrPVal
     0.82s  2.10% 62.46%      0.83s  2.12%  github.com/intuitivelabs/sipsp.skipToken
     0.82s  2.10% 64.55%      0.90s  2.30%  runtime.findObject
     0.57s  1.46% 66.01%      0.57s  1.46%  runtime.futex
     0.52s  1.33% 67.34%      4.38s 11.19%  github.com/intuitivelabs/sipsp.ParseHdrLine
     0.36s  0.92% 68.26%      0.36s  0.92%  runtime.memclrNoHeapPointers
```
![profile_x10](https://user-images.githubusercontent.com/44233689/110954239-907e1c80-8348-11eb-9a26-b12df5d310ad.png)

**Comment by poandrei** [comment]
result for the wtimer branch, alpha version, 10x speed same pcap as above:

```
Showing top 10 nodes out of 141
      flat  flat%   sum%        cum   cum%
    2600ms 14.44% 14.44%     2600ms 14.44%  runtime.futex
    1480ms  8.22% 22.67%     1690ms  9.39%  github.com/intuitivelabs/sipsp.ParseNameAddrPVal
    1170ms  6.50% 29.17%     1290ms  7.17%  runtime.cgocall
     940ms  5.22% 34.39%     1400ms  7.78%  runtime.scanobject
     860ms  4.78% 39.17%      860ms  4.78%  runtime.usleep
     720ms  4.00% 43.17%      720ms  4.00%  github.com/intuitivelabs/sipsp.skipToken
     410ms  2.28% 45.44%     4370ms 24.28%  github.com/intuitivelabs/sipsp.ParseHdrLine
     280ms  1.56% 47.00%      300ms  1.67%  github.com/intuitivelabs/sipsp.skipLWS
     280ms  1.56% 48.56%      280ms  1.56%  github.com/intuitivelabs/sipsp.skipTokenDelim
     270ms  1.50% 50.06%      330ms  1.83%  runtime.nanotime
```

![sipcm_wtimer_cpu1](https://user-images.githubusercontent.com/44233689/111846834-e97d2000-8907-11eb-90fa-01bc713f84e5.png)

**Comment by poandrei** [comment]
Fixed after migrating to wtimer (commit 1e2dc670e2e03b45714979edc976c1eb6555bd0d).
